src/simulators/tensor_network/matrix_product_state.cpp

```diff
@@ -21,6 +21,8 @@
 #include "matrix_product_state.hpp"
 #include "matrix_product_state_tensor.hpp"
 
+using namespace std;
```

Comment:
Can you remove `using namespace std;`

Reply:
Done. Also removed from svd.hpp and svd.cpp